cobbler/tftpgen.py

```diff
@@ -423,7 +424,7 @@ def get_menu_items(self, arch: Optional[enums.Archs] = None) -> dict:
         """
         return self.get_menu_level(None, arch)
 
-    def get_submenus(self, menu, metadata: dict, arch: enums.Archs):
+    def get_submenus(self, menu: Menu, metadata: dict, arch: enums.Archs):
```

Comment:
Just as a secondary note. I am aware that with Python 3.10 this just works out of the box, however many places where this will be run are still on Python 3.6 and there we might break things.